bidb/api/views.py

```diff
@@ -18,3 +24,38 @@ def submit(request):
         settings.SITE_URL,
         submission.buildinfo.get_absolute_url(),
     ), status=201 if created else 200)
+
+
+@require_http_methods(['GET'])
+def buildinfo_since(request, date):
+    try:
+        created = datetime.datetime.fromtimestamp(int(date) + 1)
+    except ValueError:
+        raise SuspiciousOperation("Date value should be an integer")
+    try:
+        limit = int(request.GET.get('limit', 100))
+        if limit <= 0:
+            raise SuspiciousOperation("Limit value should be positive")
+    except ValueError:
+        raise SuspiciousOperation("Limit value should be an integer")
+
+    buildinfo = Buildinfo.objects.filter(
+            created__gte=created
```

Comment:
Double indentation, and missing trailing comma:
```suggestion
            created__gte=created,
```